templates/problem/voting-stats.html

```diff
@@ -0,0 +1,22 @@
+<div class="vote-stats-background" id="id_vote_stats">
+    <span class="vote-stats-info">{{ _('Voting Statistics') }}</span>
+    <br/>
+    <canvas class="canvas" id="id_vote_chart"></canvas>
+    <span id="id_no_votes_error" class="vote-stats-info">{{ _('No Votes Available!') }}</span>
+    <br/>
+    <div id="id_has_votes_footer">
+        <span class="vote-stats-info">{{ _('Median Vote:') }}</span>
+        <span class="vote-stats-value" id="id_median_vote"></span>
+        <span class="vote-stats-info">{{ _('Mean Vote:') }}</span>
+        <span class="vote-stats-value" id="id_mean_vote"></span>
+        <span class="vote-stats-info">{{ _('Number of Votes:') }}</span>
+        <span class="vote-stats-value" id="id_num_of_votes"></span>
+    </div>
+</div>
+
+<a class="form-button" id="id_vote_stats_button">{{ _('Voting statistics') }}</a>
```

Comment:
Why does hovering this give the text cursor and not the pointer like it does for the other links?

Reply:
![Screenshot from 2022-03-20 22-09-56](https://user-images.githubusercontent.com/41458184/159197203-b238048f-a3f9-42ce-9af8-525135c74e26.png)

Reply:
On that note, I'm a superuser right now, where are the buttons?

Reply:
My bad, I haven't solved the problem.

Reply:
An href attribute needs to be added for it to show a pointer.

Reply:
Is there some other explicit attribute that sets the cursor? That would probably be cleaner

Reply:
Perhaps cursor: pointer on hover?